dll/win32/shell32/dialogs/recycler_prop.cpp

```diff
@@ -146,6 +147,11 @@ InitializeRecycleBinDlg(HWND hwndDlg, WCHAR DefaultDrive)
                                 defIndex = itemCount;
                                 pDefault = pItem;
                             }
+
+                            dwSize = sizeof(DWORD);
+                            RegGetValueW(HKEY_CURRENT_USER, L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Policies\\Explore", L"ConfirmFileDelete", RRF_RT_DWORD | RRF_ZEROONFAILURE, NULL, &pItem->dwConfirmDelete, &dwSize);
```

Comment:
> `L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Policies\\Explore"`

The key name should have an R at the end to match the rest of the codebase.

Reply:
Done